### PR TITLE
Reference file doc updates for IPC and Linearity

### PR DESCRIPTION
**Pull request by hbushouse**
Updated the reference file info in the `ipc` and `linearity` step docs, for the new consistent formatting of reffile docs.

### CODE REVIEW

**Comment by hbushouse** [comment]
Partially addresses #2675.

File: docs/jwst/ipc/reference_files.rst [hunk 1]
```diff
@@ -1,26 +1,56 @@
...
-The IPC deconvolution current step uses an IPC reference file.
+Reference Files
+===============
+The IPC deconvolution step uses an IPC reference file.
 
-.. include:: ../includes/standard_keywords.rst
+IPC Reference File
+------------------
+
+:RETYPE: IPC
```
Comment (stscirij) [inline docs/jwst/ipc/reference_files.rst]: Should be REFTYPE?

File: docs/jwst/linearity/reference_files.rst [hunk 2]
```diff
@@ -1,28 +1,53 @@
...
 ====================
 
-The linearity correction step uses a LINEARITY reference file.
+The ``linearity`` step uses a LINEARITY reference file.
 
-.. include:: ../includes/standard_keywords.rst
+LINEARITY Reference File
+-------------------------
+
+:RETYPE: LINEARITY
```
Comment (stscirij) [inline docs/jwst/linearity/reference_files.rst]: Should be REFTYPE?

File: docs/jwst/ipc/reference_files.rst (hunk 1 above)
Comment (hbushouse) [reply docs/jwst/ipc/reference_files.rst]: Yep.  That's a typo that's in my template that I've been using to make updates for _all_ the steps and you're the first one to spot it. Good eye!

**Pull request merged**